package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc",
     "watch": "tsc -w",
     "nodewatch": "nodemon index.js",
-    "start": "node dist/index.js",
+    "start": "node dist/cmdutil.js",
```

Comment:
@User1m not sure why you changed the start script to the cmdutil.js. That file was actually for running the connector from the command line, which I ended up deprecating. Did you mean to include this in your PR?

Reply:
@ryanvolum ohh. I assumed index was a left over reference bc there's no compiled index.js file. But I can remove this from the PR.